Automattic/wp-calypso
# Starter can upgrade to Premium, and Pro Monthly can upgrade to Business or Ecommerce Monthly

#### Proposed Changes

* Ref pdgrnI-1sh-p2#comment-2559
* Starter can upgrade to Premium
* Pro monthly can upgrade to Business Monthly or Ecommerce Monthly
* Pro Monthly can upgrade to Business and eCommerce yearly plans

#### Testing Instructions

* Go to a site that has Starter Plan. Verify that you can see the upgrade buttons, and that you're able to checkout and upgrade your site to Premium.
* Go to a site with Pro monthly. Verify the same for Business Monthly and Ecommerce monthly
* Verify that you can upgrade a Pro Monthly site to Business or eCommerce yearly plans.
![Screenshot 2022-07-19 at 09 56 48](https://user-images.githubusercontent.com/52675688/179699575-c3bce1a4-231d-446f-b8f8-c7a4e01a5c12.png)
![Screenshot 2022-07-19 at 09 58 01](https://user-images.githubusercontent.com/52675688/179699596-56c3f893-ffb9-41d5-b4ef-09ed4e7d5a25.png)
![Screenshot 2022-07-19 at 10 08 01](https://user-images.githubusercontent.com/52675688/179700366-33c8b462-05d5-4168-974b-37d006f52aba.png)




#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.


Related to #
